[BUG] gradle precommit is broken due to transportClientRatio

**Describe the bug**
Gradle precommit on local desktop is broken

```
/home/ubuntu/sarat-opensearch/server/src/test/java/org/opensearch/cluster/routing/MovePrimaryFirstTests.java:23: error: cannot find symbol
@OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0, transportClientRatio = 0)
                                                                                                    ^
  symbol:   method transportClientRatio()
  location: @interface ClusterScope
```

Not sure why GHA action for precommit is going through. But its quite reproducible on local machine.
**To Reproduce**
Steps to reproduce the behavior:
1. Checkout OpenSearch
2. Run `./gradlew precommit`

**Host/Environment (please complete the following information):**
 - OS: Mac OS BigSur, Ubuntu (Linux)

--- a/server/src/test/java/org/opensearch/cluster/routing/MovePrimaryFirstTests.java
+++ b/server/src/test/java/org/opensearch/cluster/routing/MovePrimaryFirstTests.java
@@ -20,7 +20,7 @@ import java.util.concurrent.CountDownLatch;
 
 import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertAcked;
 
-@OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0, transportClientRatio = 0)
+@OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0)
 @ThreadLeakScope(ThreadLeakScope.Scope.NONE)
 public class MovePrimaryFirstTests extends OpenSearchIntegTestCase {
 

